.github/stale.yml

```diff
@@ -0,0 +1,38 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - security
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
```

Comment:
Shouldn't such issue stay open?

Reply:
I would say if an issue is going to be fixed, there should be some activity on it. We should to look at which issues are being marked stale and make sure there is activity so they don't actually get closed if they are important. I'd suggest we start here and if we find this is too aggressive, we can change it.